community/blog/2022-12 Arcus Web API v1.7, a release with improvements like never before/README.md

```diff
@@ -0,0 +1,50 @@
+# Arcus Web API v1.7, a release with improvements like never before
+Arcus Web API v1.7 brings improvements to the table that make it the most production-ready release to date. It fully embraces Microsoft technology like never before.
+
+## W3C HTTP correlation for automatic Microsoft dependency tracking
+Arcus already had service-to-service correlation between Arcus components. A HTTP request was correlated into an Azure Service Bus message and so on. Each component was linked together in a parent-child relationship that can be seen in Azure Application Insights. This was already there; what was not were Microsoft dependencies. When discarding Arcus observability technology and only using Microsoft Application Insights service registration, you get automatic dependency tracking for Microsoft dependencies. This includes SQL operations, HTTP requests, Azure Service Bus messages and lots more. It was a real argument against Arcus, until the Web API v1.7 release.
+
+Microsoft uses activities behind the scenes to set and get hold of the correlation of the application. Arcus used a dedicated registered service to do that. Starting from this release, both are used. Arcus makes sure that it sets and uses the correlation available in the current activity. This means that both any tracked Arcus dependencies as well as any Microsoft dependencies use the same information. This results in a system where both are working together instead of against each other. Any Microsoft dependencies will be tracked automatically in an Arcus-configured Web API and any non-Microsoft dependencies can be easily tracked with our dependency logger extensions. Best of both worlds, as they say.
```

Comment:
```suggestion
Microsoft uses activities behind the scenes to set and get hold of the correlation of the application. Arcus used a dedicated registered service to do that. Starting from this release, both are used. Arcus makes sure that it sets and uses the correlation available in the current activity. This means that any tracked Arcus dependencies as well as any Microsoft dependencies use the same information. This results in a system where both are working together instead of against each other. Any Microsoft dependencies will be tracked automatically in an Arcus-configured Web API and any non-Microsoft dependencies can be easily tracked with our dependency logger extensions. Best of both worlds, as they say.
```